src/views/connections/ConnectionForm.vue

```diff
@@ -722,7 +759,13 @@ export default class ConnectionForm extends Vue {
         this.$message.error(msgError)
         this.$log.error(msgError)
       }
-    })
+    }
+  }
+
+  private handleActionCommand(command: string) {
+    if (command === 'handleSave') {
+      this.handleSave('save')
+    }
```

Comment:
It's recommended to use `save` for commands, as `handleSave` sounds more like a function name.

Consider future scalability.

```suggestion
  private handleActionCommand(command: 'save') {
    if (command === 'save') {
      this.handleSave('save')
    }
```